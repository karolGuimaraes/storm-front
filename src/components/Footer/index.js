import React from 'react';
import styles from './styles.module.css';
import RadioButtonCheckedIcon from '@material-ui/icons/RadioButtonChecked';
import TvIcon from '@material-ui/icons/Tv';
import UpdateIcon from '@material-ui/icons/Update';

export default function Footer () {
  return (
    <div>
      <footer className={styles.footer}>
        <div>
          <div className={styles.divOnAir}>
            <RadioButtonCheckedIcon className={styles.iconOnAir}/>
            <span>No Ar</span>
          </div>
          <div className={styles.info}>
            <TvIcon className={styles.icon}/>
            <span>ENCONTRO - 10:00</span>
          </div>
          <div className={styles.info}>
            <UpdateIcon className={styles.icon}/>
            <span>Última atualização em 10:28</span>
          </div>
        </div>

        <div className={styles.footerRight}>
          <span>Quinta, 30 de Maio de 2019</span>
          <span className={styles.hour}>// 10:32:42</span>
        </div>
      </footer>
    </div>
  )
}