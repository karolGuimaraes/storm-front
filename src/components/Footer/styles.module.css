.footer {
  display: flex;
  justify-content: space-between;
  align-items: center;
  background-color: #333333;
  color: #E9E9E9;
  width: 100%;
  margin-top: 15px;
  font-size: 14px;
  padding: 10px 0 10px 0;
  position: fixed;
  bottom: 0;

}

.headerLeft {
  display: flex;
  justify-content: center;
  align-items: center;
}

.headerRight {
  display: flex;
  justify-content: center;
  align-items: center;
  margin-right: 10px;
}

.divAir {
  color: #FF8700;
  font-family: 'Audiowide';
  font-weight: 400;
  margin-left: 10px;
  margin-right: 45px;
  padding: 10px;
}

.iconAir {
  display: inline-block;
  vertical-align:middle;
  margin-right: 5px;
  width: 16px;
  height: 16px;
}

.info {
  margin-right: 30px;
  padding: 10px;
}

.icon {
  display: inline-block;
  vertical-align:middle;
  margin-right: 5px;
  color: #666666;
  width: 16px;
  height: 16px;
}

.hour {
  color: #FF8700;
  margin-left: 5px;
  font-size: 24px;
  font-weight: 400;
}