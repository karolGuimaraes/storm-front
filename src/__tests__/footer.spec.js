import React from 'react';
import { create } from "react-test-renderer";
import { shallow } from 'enzyme';
import Footer from '../components/Footer';

describe("Footer Component Test", () => {
  test("renders correctly", () => {
    const footer = create(<footer />);
    expect(footer.toJSON()).toMatchSnapshot();
  });
});