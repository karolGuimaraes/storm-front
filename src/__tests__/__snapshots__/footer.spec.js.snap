// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component Test renders correctly 1`] = `<footer />`;

exports[`Footer Component Test should  1`] = `<footer />`;

exports[`Test Footer Component should  1`] = `<footer />`;
